README.md

```diff
@@ -1,17 +1,16 @@
 # DnDeep-backend
 The backend part of DnDeep
 
-to create a Character send a POST request to http://localhost:3000/characters/ (if on local host) with a body similar to what is seen bellow:
+to create a Character send a POST request to http://{localhost}:3000/characters/ (if on local host) with a body similar to what is seen bellow:
```

Comment:
This is my own mistake, codacy wants to have angle brackets on each side like this <Link> on all literal urls

Reply:
I missunderstood and so made it curly brackets instead which should probably be removed

Reply:
I've pushed a new commit with better Markdown :)